### PR TITLE
[CP] 3.7 Cherry picks

### PR DESCRIPTION
**Pull request by jturel**
(cherry picked from commit b70dde1807e6b96c92d5f893d67e1172fd8a9722)

### CODE REVIEW

**Comment by ekohl** [comment]
Should we also update candlepin to [5.1.1](https://github.com/theforeman/puppet-candlepin/blob/master/CHANGELOG.md#511-2018-07-11)?

**Comment by jturel** [comment]
@ekohl updated!

File: Puppetfile.lock [hunk 1]
```diff
@@ -45,13 +45,13 @@ FORGE
...
-    puppet-extlib (2.0.1)
+    puppet-extlib (2.1.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.6.0)
-    puppet-mongodb (2.2.1)
+    puppet-mongodb (2.2.2)
       puppetlabs-apt (< 5.0.0, >= 2.1.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.13.0)
     puppet-staging (3.2.0)
-    puppetlabs-apache (3.1.0)
+    puppetlabs-apache (3.2.0)
```
Comment (ekohl) [inline Puppetfile.lock]: In foreman-installer we've pinned to `3.1.0` so I'd prefer to keep these in sync.

File: Puppetfile.lock [hunk 2]
```diff
@@ -60,12 +60,16 @@ FORGE
       puppetlabs-stdlib (< 5.0.0, >= 4.13.1)
     puppetlabs-firewall (1.12.0)
       puppetlabs-stdlib (< 5.0.0, >= 4.0.0)
-    puppetlabs-hocon (1.0.0)
-    puppetlabs-inifile (2.2.2)
+    puppetlabs-hocon (1.0.1)
```
Comment (ekohl) [inline Puppetfile.lock]: same for hocon an inifile

File: Puppetfile.lock (hunk 1 above)
Comment (jturel) [reply Puppetfile.lock]: I followed the documented instructions to pin new versions. Should this have changed? I didn't do it deliberately.

**Comment by jturel** [comment]
@ekohl better now?

File: Puppetfile.lock (hunk 1 above)
Comment (ekohl) [reply Puppetfile.lock]: No, it's just a nasty process where the tools aren't exactly designed to do what we want. My goal is to closer align to the tooling but that comes down to merging katello-installer into foreman-installer and a rather big change.

**Comment by jturel** [comment]
[test]

**Comment by jturel** [comment]
[test continuous-integration/travis-ci/pr]

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**